Subject: [PATCH] [BUGFIX] Skip deleted content elements
---
Index: typo3/sysext/dashboard/Classes/Widgets/LatestChangedPagesWidget.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Classes/Widgets/LatestChangedPagesWidget.php b/Classes/Widgets/LatestChangedPagesWidget.php
--- a/Classes/Widgets/LatestChangedPagesWidget.php	(revision eca0abc5c3f4f4a9bba791f1e5b547d5a5ea5aa2)
+++ b/Classes/Widgets/LatestChangedPagesWidget.php	(revision fedcd31cdcfa04d3e5aaade280fdd828a4ef2526)
@@ -174,8 +174,13 @@
             ->executeQuery()
             ->fetchAssociative();

+        if (!$contentRecord) {
+            // Content element has been deleted
+            return 0;
+        }
+
         $queryBuilder = $this->getQueryBuilderForPages();
-        if ($contentRecord !== false && (int)$contentRecord['sys_language_uid'] > 0) {
+        if ((int)$contentRecord['sys_language_uid'] > 0) {
             return $queryBuilder
                 ->select('uid')
                 ->from('pages')
